src/shipit_code_coverage/tests/conftest.py

```diff
@@ -120,3 +131,21 @@ def grcov_uncovered_function_artifact():
 def jsvm_uncovered_function_artifact():
     with generate_coverage_artifact('jsvm_uncovered_function.info') as f:
         yield f
+
+
+@pytest.fixture(scope='session')
+def mock_get_commit(MERCURIAL_COMMIT):
+    responses.add(
+        responses.GET,
+        'https://api.pub.build.mozilla.org/mapper/gecko-dev/rev/hg/{}'.format(MERCURIAL_COMMIT),
+        body='{} 0d1e55d87931fe70ec1d007e886bcd58015ff770'.format(MERCURIAL_COMMIT),
```

Comment:
The mercurial commit should be the second in the responses, see https://api.pub.build.mozilla.org/mapper/gecko-dev/rev/hg/0d1e55d87931fe70ec1d007e886bcd58015ff770 for example.

Reply:
Done